src/main/60-eks.tf

```diff
@@ -0,0 +1,71 @@
+resource "aws_iam_policy" "additional" {
+  name = "${var.app_name}-fargate-additional"
+
+  policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      {
+        Action = [
+          "ec2:Describe*",
+        ]
+        Effect   = "Allow"
+        Resource = "*"
+      },
+    ]
+  })
+}
+
+module "eks" {
+  source  = "terraform-aws-modules/eks/aws"
+  version = "19.10.0"
+
+  cluster_name    = "${var.app_name}-eks-${var.env}"
+  cluster_version = "1.24"
```

Comment:
You can use the variable you have declared in the variables file

Reply:
Is meant to pin the version, but I'll set the variables